e1.6-1.14/src/App.js

```diff
@@ -0,0 +1,18 @@
+
+import Header from './header'
+import Buttons from './buttons'
+
+
+const App = () => {
+  
+
+  return (
+    <div>
+      <Header/>
+      <Buttons/>  
+     
+    </div>
+  )
+}
+
+export default App
```

Comment:
@chrisOc93 if you use VS Code set this - https://stackoverflow.com/a/54270768 to automatically add a new line to the end when saving a file